rpm/generic/zfs.spec.in

```diff
@@ -457,6 +457,10 @@ systemctl --system daemon-reload >/dev/null || true
 %config(noreplace) %{_sysconfdir}/%{name}/zpool.d/*
 %config(noreplace) %{_sysconfdir}/%{name}/vdev_id.conf.*.example
 %attr(440, root, root) %config(noreplace) %{_sysconfdir}/sudoers.d/*
+%if 0%{?_pam}
+%{_pammoduledir}/*
+%{_pamconfigsdir}/*
+%endif
```

Comment:
Distributions might possibly put these in separate subpackages.

Reply:
so you would suggest to move them to a `libpam_zfs_key` and `pam_zfs_key` package in a new `files -n`-section